sources/lists/lineSeperated.go

```diff
@@ -60,6 +59,11 @@ func (m *lineSeperatedList) FetchData(entryMap list.BlacklistedEntryMap, errorsO
 		if len(line) == 0 {
 			continue
 		}
+		//skip commented lines
+		if line[0] == '#' {
```

Comment:
Could we add a line to the lineSeparated_test.go to document/ ensure this functionality?

Reply:
Added.